compiler/src/iree/compiler/Dialect/Util/Analysis/Constant/OpOracle.cpp

```diff
@@ -61,8 +61,8 @@ bool isLegalConstExprType(Type t) {
     // support, but for now the consteval JIT has interop limitations. Lift
     // this restriction when the JIT interops for all types.
     auto bitWidth = t.getIntOrFloatBitWidth();
-    return bitWidth == 1 || bitWidth == 8 || bitWidth == 16 || bitWidth == 32 ||
-           bitWidth == 64;
+    return bitWidth == 1 || bitWidth == 4 || bitWidth == 8 || bitWidth == 16 ||
```

Comment:
I'd not want to special case 4 here as it's really any power-of-two value up to 64 bits (so 2 should be added, at which point we can do `isPowerOf2(bitWidth) && bitWidth <= 64`